src/lib/SCEditor.js

```diff
@@ -2250,6 +2250,18 @@ export default function SCEditor(original, userOptions) {
 		updateActiveButtons();
 	};
 
+	/**
+	 * Returns list of tags from the WYSIWYG editor
+	 *
+	 * @param {string} tagName
+	 * @function
+	 * @name execCommand
+	 * @memberOf SCEditor.prototype
+	 */
+	base.wysiwygGetTags = function (tagName) {
+		return wysiwygDocument.getElementsByTagName(tagName);
```

Comment:
This returns a **live** ordered collection of **Elements** by tag name.
Either be explicit about it in its documentation or change the code into something most programmers would expect from it

Reply:
I'll check other methods, maybe it will be easier to just inject the HTML tags instead of using execCommand